### DISCUSSION TITLE
Best way to secure a prefect server (core)

### DISCUSSION THREAD

**Discussion by fpun** [PrefectHQ/prefect#3613] [Q&A]
I read through the doc but couldn't find a way to restrict access to a prefect server instance. I understand this is available in cloud but is there any built in feature in the core version as well? 
If not, any idea what would be the most straightforward way to add restrictions?

**Comment by cicdw** [comment]
Hi @fpun - as a general rule we avoid making any recommendations about how to implement security for Server / Core, as this is highly dependent on your organizations rules and depends on how you choose to deploy it (i.e., just putting some security into code isn't enough, it needs to also be networked and deployed securely).  

That being said, other people in the community may have thoughts / opinions here so they can still chime in!

**Reply by fpun** [reply]
Thank you @cicdw, I was hoping for a simple login system on the UI that would establish a secure connection with the API but I understand your positioning. If there is any best practice from the community, I would love to know about them!
